### PR TITLE
Setup unit and integration tests

### PR DESCRIPTION
**Pull request by stanislasbonifetto**

### CODE REVIEW

File: hk-kafka-topics-connector.cabal [hunk 1]
```diff
@@ -4,7 +4,7 @@ cabal-version: 1.12
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: 4c217a6503c5728ea1742e2f8f0a9fe124ac12f4acc7234ee0bb2886afff5bc0
+-- hash: b125006bf000f01fbd70c91f506dc1fab05fd452c04538c3962ffa9e5b5afdb7
```
Comment (stanislasbonifetto) [inline hk-kafka-topics-connector.cabal:7]: @demaledetti is the `*.cabal` file generate from stack?
If it is that the case, should not be better not include in the repo?

File: stack.yaml.lock [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```
Comment (stanislasbonifetto) [inline stack.yaml.lock:1]: @demaledetti is the `stack.yaml.lock` file generate from stack?
If it is that the case, should not be better not include in the repo?

File: hk-kafka-topics-connector.cabal (hunk 1 above)
Comment (demaledetti) [reply hk-kafka-topics-connector.cabal:7]: For the record, we decided to go with this:
https://tech.fpcomplete.com/blog/storing-generated-cabal-files
Let's see how quickly we'll get bitten by this:
https://github.com/sol/hpack/issues/380

File: stack.yaml.lock (hunk 2 above)
Comment (demaledetti) [reply stack.yaml.lock:1]: For the record, we decided to go with this:
https://groups.google.com/forum/m/#!topic/haskell-stack/xeTN_3UNwE4

**Pull request merged**